solr/core/src/test/org/apache/solr/search/TestCpuAllowedLimit.java

```diff
@@ -61,7 +65,12 @@ private static Path createConfigSet() throws Exception {
   }
 
   @BeforeClass
-  public static void setup() throws Exception {
+  public static void setupClass() throws Exception {
+    // Using NoMergePolicy and 100 commits we should get 100 segments (across all shards).
+    // At this point of writing MAX_SEGMENTS_PER_SLICE in lucene is 5, so we should be
```

Comment:
(not yet merged) https://github.com/apache/lucene/pull/13508 includes making Lucene's `IndexSearcher.MAX_SEGMENTS_PER_SLICE` public.

Reply:
> (not yet merged) [apache/lucene#13508](https://github.com/apache/lucene/pull/13508) includes making Lucene's `IndexSearcher.MAX_SEGMENTS_PER_SLICE` public.

I see, though still final. Until such time as we override [IndexSearcher#slices(List<LeafReaderContext>)](https://github.com/apache/lucene/blob/7b421d75c9ee829c9745f0f7ad8f6282535cbe5d/lucene/core/src/java/org/apache/lucene/search/IndexSearcher.java#L324) this is still a constant.